lib/chef/knife/cloud/vra_service.rb

```diff
@@ -67,8 +67,9 @@ def create_server(options = {})
           servers.first
         end
 
-        def delete_server(instance_id)
-          server = get_server(instance_id)
+        def delete_server(deployment_id)
+          deployment = get_deployment(deployment_id)
+          server = deployment.resources.select(&:vm?).first
```

Comment:
Can we call this method `get_server`. Looking at this method seems like it is doing the same thing.

Reply:
I think the method name `get_deployment` makes sense as it is not returning server. we are fetching server from it.
`get_server` method exists below

Reply:
Hi @sanjain-progress,

Even though `get_server` method has these two same lines, I think we cannot use it here. Since we need the deployment object later in the method, the get_server method wont return that.